src/collections/_documentation/clients/ruby/context.md

```diff
@@ -53,7 +53,7 @@ Raven.user_context(
   username: current_user.username, # "foo"
 
   # the actor's IP address, if available
-  ip_address: request.ip # '127.0.0.1'
+  ip_address: request.remote_ip # '127.0.0.1'
```

Comment:
While rails does offer `request.remote_ip` the standard Rack interfaces do not. Adding this would result in broken examples for all non-rails use cases.

Reply:
D'uh. You're absolutely correct. I didn't realize, that this piece of documentation was targeted at all ruby users.